server/src/gateway/dto/cursor.data.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  IsDefined,
+  IsNumber,
+  IsString,
+} from 'class-validator';
+
+export class CursorMoveData {
+  @IsDefined()
+  @IsNumber()
+  x: number;
+
+  @IsDefined()
+  @IsNumber()
+  y: number;
+
+  @IsDefined()
+  @IsString()
+  color: string;
```

Comment:
is the color in hex?

Reply:
doesn't matter

Reply:
I'm not sure what the final color will be; we need to discuss this later. For now, I've only added this color on the client side. We might either go with a random color approach or save a different color property in the database, but I'm not certain yet.

Reply:
Okay, so i propose to create separate task for colors